docs/proposals/dynamic-openshift-settings-configmaps.md

```diff
@@ -0,0 +1,55 @@
+# Changing OpenShift settings dynamically through ConfigMaps
+
+## Problem
+
+Currently, there are settings contained withing the master-config.yaml file that
+are of value to an unpriviledged user. Specifically, the `autoProvisionEnabled`
+boolean under `jenkinsPipelineConfig` is only available to users with the
+ability to write to the master-config.
+
+## Design proposal:
+
+- In each project there can exist a ConfigMap annotated with
+  'build.openshift.io/configuration'
+
+- The ConfigMap will contain the key/value pair of the setting to be overridden
+  (key) and the value to use.
+
+- Only certain settings would be covered under this method.
```

Comment:
the point of my comment was that this line should be changed, something more like:

"Some cluster settings will be overridable at the project level using this method, and other project specific settings can be introduced via the configmap as well."